pages/api/Schema/reports/designationAccounts/designationAccounts.graphql

```diff
@@ -1,10 +1,30 @@
 extend type Query {
-  designationAccounts(accountListId: ID!): [DesignationAccount!]!
+  designationAccounts(accountListId: ID!): [DesignationAccountsGroup!]!
+}
+
+extend type Query {
+  activeDesignationAccount(
+    accountListId: ID!
+    designationAccountId: ID!
```

Comment:
Doesn't this endpoint need to take whether the designation account is active or not?
```suggestion
    designationAccountId: ID!
    active: Boolean!
```